shell/platform/windows/fixtures/main.dart

```diff
@@ -33,3 +35,28 @@ void verifyNativeFunctionWithReturn() {
   bool value = signalBoolReturn();
   signalBoolValue(value);
 }
+
+@pragma('vm:entry-point')
+void drawHelloWorld() {
+  ui.PlatformDispatcher.instance.onBeginFrame = (Duration duration) {
+    final ui.ParagraphBuilder paragraphBuilder = ui.ParagraphBuilder(ui.ParagraphStyle())
+      ..addText('Hello world');
+    final ui.Paragraph paragraph = paragraphBuilder.build();
+
+    paragraph.layout(const ui.ParagraphConstraints(width: 800.0));
+
+    final ui.PictureRecorder recorder = ui.PictureRecorder();
+    final ui.Canvas canvas = ui.Canvas(recorder);
+
+    canvas.drawParagraph(paragraph, ui.Offset.zero);
+
+    final ui.Picture picture = recorder.endRecording();
+    final ui.SceneBuilder sceneBuilder = ui.SceneBuilder()
+      ..addPicture(ui.Offset.zero, picture)
+      ..pop();
+
+    ui.window.render(sceneBuilder.build());
```

Comment:
I wonder if we should put a signal before this line, then in the C++ portion of the test, check that the callback hasn't been called.

Reply:
That's a good idea, will add!

Reply:
Added check